<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateDialog</class>
 <widget class="QDialog" name="updateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改信息</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0" rowspan="2" colspan="2">
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>商品名称：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>编号：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>总数量：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>进价：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>售价：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>生产厂家：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>生产日期：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="dateEdit"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_8"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>保质期(月)：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>updateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>updateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
